components/blocks/Run/Run.vue

```diff
@@ -13,11 +13,11 @@
     </div>
     <div class="run__info">
       <p>{{ time }}</p>
-      <div class="hidden md:flex flex-row md:flex-grow">
+      <div class="hidden flex-row md:flex md:grow">
         <p>{{ platform }}</p>
       </div>
       <div
-        class="flex flex-row opacity-75 gap-x-5 md:opacity-100 justify-end items-center"
+        class="flex flex-row gap-x-5 justify-end items-center opacity-75 md:opacity-100"
```

Comment:
Why the rearrangement? Does order matter, or is this simply preference? Ditto for the rest

Reply:
Ah sweet. All good then.